### PR TITLE
docs(client): working on new doc

### PR DESCRIPTION
**Pull request by javadkh2**

### CODE REVIEW

File: packages/libs/client/NEW_README.md [hunk 1]
```diff
@@ -0,0 +1,2061 @@
...
+**Note**: In many cases, you'll need to store the result of this function to
+fetch the result of the request.
+
+#### Client: send
+
+`send` is a deprecated alias for the `submit` function with the same interface.
+
+#### Client: submitOne
+
+`submitOne` is an alias for the first overload of the `submit` function:
```
Comment (lsgunnlsgunn) [inline packages/libs/client/NEW_README.md:1201]:
```suggestion
The `submitOne` function is the same as submitting one transaction using the `submit` function.
For example:
```

**Review by lsgunnlsgunn — COMMENTED** [review]
Mostly minor changes or nit picky things.
Stopped at line 1231.
I had a few questions in there and a little bit of creative license.
If you're okay with committing the changes. I'd be fine with pulling this into the docs this week and iterating further as a follow-on task

File: packages/libs/client/NEW_README.md [hunk 2]
```diff
@@ -0,0 +1,2061 @@
...
+| txList    | ICommand[] | The command objects ready to submit |
+
+**Note**: In many cases, you'll need to store the result of this function to
+fetch the result of the request.
+
+#### Client: send
+
+`send` is a deprecated alias for the `submit` function with the same interface.
+
+#### Client: submitOne
```
[the remark this answers is not in the document]
Comment (javadkh2) [reply packages/libs/client/NEW_README.md:1199]: Do you mean remove these lines? then don't we need documentation for Client: send and Client: submitOne

File: packages/libs/client/NEW_README.md [hunk 3]
```diff
@@ -0,0 +1,2061 @@
...
+      'alice',
+      'bob',
+      readKeyset('bob-guard'),
+      { decimal: '1.1' },
+    ),
+  )
+  .addKeyset('bob-guard', 'keys-all', 'bob-public-key');
+```
+
+Transfer create as string code
```
[the remark this answers is not in the document]
Comment (javadkh2) [reply packages/libs/client/NEW_README.md:435]: Yes its the same.
I wanted to show you can use simple string or use `Pact.modules`.

File: packages/libs/client/NEW_README.md [hunk 4]
```diff
@@ -0,0 +1,2061 @@
...
+
+**note**: This function uses `quicksign` protocol.
+
+```typescript
+createSignWithChainweaver(options:{ host?: string }): ISignFunction
+```
+
+| Parameter | Type              | Description                                                           |
+| --------- | ----------------- | --------------------------------------------------------------------- |
+| option    | { host?: string } | option including host URL default `{ host: 'http://127.0.0.1:9467' }` |
```
[the remark this answers is not in the document]
Comment (javadkh2) [reply packages/libs/client/NEW_README.md:752]: yes, this option is for passing different host url than the default one. 
we wrote it as as object for more flexibility; if we want to add more options to it later.

**Comment by javadkh2** [comment]
Lisa already used the doc in this PR ; so I closed it

**Pull request closed without merging**